lib/elrondex/sc/pair_sc.ex

```diff
@@ -1,21 +1,38 @@
 defmodule Elrondex.Sc.PairSc do
   alias Elrondex.{Sc, Transaction, Account, REST, ESDT, Pair, Network}
 
+  @doc """
+  Swaps a certain token amount with fixed amount value on input and minimum amount value on output.
+
+  ## Arguments
+    * `account` - An account's struct
+    * `pair` -  A pair struct 
+    * `token_in` - Token identifier that we provide on input for swap operation
+    * `value_in` - Fixed amount value for input token
+    * `token_out` - Token identifier that we provide on output. The swap operation will return given token into account
+    * `value_out_min` - Minimum amount for output token that is required for a successful swap operation
+
+  ## Examples
+
+      iex> Elrondex.Test.Bob.swap_1_egld_to_min_100usdc
```

Comment:
insert full call here with `Elrondex.Test.Bob.account()` and `Elrondex.Test.Pair.wegld_usdc_pair()`